docs/admin/authorization.md

```diff
@@ -53,7 +53,7 @@ A request has the following attributes that can be considered for authorization:
   - what resource is being accessed (for resource requests only)
   - what subresource is being accessed (for resource requests only)
   - the namespace of the object being accessed (for namespaced resource requests only)
-  - the API group being accessed (for resource requests only)
+  - the API group being accessed (for resource requests only); the [core API group](../api.md#api-groups) is identified by the empty string
```

Comment:
I'm confused by "the empty string"--was this empty string mentioned before? Or do you need to supply an empty string for the core API group?